spec/factories/events.rb

```diff
@@ -0,0 +1,33 @@
+FactoryGirl.define do
+  factory :event do
+    title "Blessings wedding"
+    description "Happy day of joy celebration happinness smiles."
+    location "Beach side"
+    start_date Time.now
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.